property-files-loader/src/main/java/com/msm/property/file/loader/readers/PropertyFilesReader.java

```diff
@@ -0,0 +1,90 @@
+package com.msm.property.file.loader.readers;
+
+import com.msm.property.file.loader.annotations.PropertiesResource;
+import com.msm.property.file.loader.annotations.PropertyName;
+import org.reflections.Reflections;
+
+import java.io.*;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author riste.jovanoski
+ * @since 6/13/2017
+ */
+public class PropertyFilesReader {
+
+    private static final Reflections reflections = new Reflections();
+    private static final String separator = File.separator;
+    private static final String propertySplitter = "=";
+    private static final String userDir = System.getProperty("user.dir");
+
+    /**
+     * Finds all classes annotated with @{@link PropertiesResource} and creates instances populated with the values
+     * of the corresponding file. Returns a Map where the key is the class and the values is instance of
+     * that class populated with the values
+     * @return Map
+     */
+    public static Map<Class, Object> loadPropertyFiles() {
```

Comment:
Avoid methods which do everything. Decompose the method in a small methods to improve code readability